ext/dl_test/tests/skip.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Check that the dl_test extension is built. We don't use the --EXTENSIONS--
+// section because we want to load the extension with dl().
+
+if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+    $path = ini_get('extension_dir') . DIRECTORY_SEPARATOR . 'php_dl_test.dll';
+} else {
+    $path = ini_get('extension_dir') . DIRECTORY_SEPARATOR . 'dl_test.so';
```

Comment:
Do we have to cater to macOS which uses `.dylib` as far as I know?

Reply:
I checked because I was not sure. The PHP build appears to generate `.so` files. The CI looks ok too (no mention of skipping or failing these tests).